Podfile

```diff
@@ -22,9 +22,9 @@ def lib_main
 end
 
 def lib_dev
-    pod 'AEPCore', :git => 'https://github.com/adobe/aepsdk-core-ios.git', :branch => 'dev-v3.7.3'
```

Comment:
the specified branch could be removed right? the core dependency could be 
```
pod 'AEPCore', '~> 4.0'
```

Reply:
this is just for the `lib_dev` group.  the podfile isn't actually using this group at all right now, only using `lib_main`